### ISSUE TITLE
Server failing to load, suggesting TOPAddons fault ?

### ISSUE THREAD

**Issue by JonSar-GitHub** [DrManganese/TOPAddons#111]
**Describe the bug**
Launching a new FTB OCeanblock server in a linux terminal windows. Launching server fails. Logs attached. 

**To Reproduce**
Fresh install of FTB Oceanblock from FTB site
Linux UbuntuMATE
Launch via start.sh in terminal and launching the server fails

**Expected behavior**
Expect the server to start up and run. 

**Screenshots**
N/A - log file included

**Mod environment (please complete the following information):**
Forge: 36.2.20
theoneprobe-1.16-3.1.4.jar
topaddons-1.16.5-2.2.0-beta.jar

**Additional context**
https://pste.ch/modebixivu.md

**Comment by DrManganese** [comment]
It seems like awt is not in your Java installation. After some googling it seems like a reinstall of openjre (not headless) would fix it.

**Comment by deckstose** [comment]
That is true. But most servers don't have the non-headless jre installed. I think it would be better if the mod checks client/server side before loading the shared libraries (I guess the mod does from reading the error log, I haven't checked the source code yet).

**Comment by DrManganese** [comment]
You have a good point about servers with the headless jre, I shouldn't expect that. I have removed references to awt in the 1.18 branch, I will make see about backporting to 1.16 too.
